lib/tictactoe.rb

```diff
@@ -1,6 +1,17 @@
 class Tic_Tac_Toe
   def run_app()
     puts("Welcome to Tic-Tac-Toe")
-    "Welcome to Tic-Tac-Toe"
```

Comment:
Let's call `puts show_empty_board` here instead of in the runner

Reply:
👍🏽